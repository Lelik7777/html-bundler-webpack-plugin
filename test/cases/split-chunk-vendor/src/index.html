<html>
<head>
  <link href="./style.scss" rel="stylesheet">
  <script src="@test-fixtures/lorem" defer="defer"></script>
</head>
<body>
  <h1>Hello World!</h1>
  <script src="./main"></script>
</body>
</html>