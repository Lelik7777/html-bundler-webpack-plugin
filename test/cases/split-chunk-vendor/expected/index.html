<html>
<head>
  <link href="/assets/css/style.026fd625.css" rel="stylesheet">
  <script src="/assets/js/runtime.a5bd90e1.js" defer="defer"></script>
<script src="/assets/js/vendors.9326a07a.js" defer="defer"></script>
<script src="/assets/js/fixture-lorem.dcb3b569.js" defer="defer"></script>
</head>
<body>
  <h1>Hello World!</h1>
  <script src="/assets/js/main.320e61d6.js"></script>
</body>
</html>